src/bin/packageJson.test.ts

```diff
@@ -0,0 +1,39 @@
+import { describe, expect, it, vi } from "vitest";
+
+import packageData from "../../package.json" assert { type: "json" };
+import { getVersionFromPackageJson } from "./packageJson.js";
+
+const mockReadFileSafeAsJson = vi.fn();
+
+/* 
+Mock the readFileSafeAsJson function that is called in getVersionFromPackageJson().
+
+Modeled on readDefaultsFromDevelopment.test.ts line 7
+*/
+vi.mock("../shared/readFileSafeAsJson", () => ({
```

Comment:
Aha! Found it - this has to be the same as the file import. That includes an extension.

```suggestion
vi.mock("../shared/readFileSafeAsJson.js", () => ({
```